joomla/joomla-cms
# Another subform fix for IE compatibility

Pull Request for Issue #25194 
Simplified version of #25200 from @ventsonge

### Testing Instructions

Apply patch and make sure the subform works as before, plus it works in IE.

@Quy instruction for IE testing:

Use IE11

Go System > Global Configuration > Users > Email Domain Options
Template should not be displayed in the bottom left corner.

Go Extensions > Plugins > System - Redirect
Template should not be displayed at the bottom initially.

Additionally, a new rows should be added without an issue.
